File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Python/Parte1/triangulo.py
from figura import Figura

class Triangulo(Figura):
    def __init__(self, base, altura):
        super().__init__(base)
        self.dimension2 = altura

    def calcular_area(self):
        return 0.5 * self.dimension1 * self.dimension2

    def calcular_perimetro(self):
        return self.dimension1 * 3

File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Java/Parte1/Triangulo.java
public class Triangulo extends Figura {
    private double dimension2;

    public Triangulo(double base, double altura) {
        super(base);
        this.dimension2 = altura;
    }

    @Override
    public double calcularArea() {
        return 0.5 * dimension1 * dimension2;
    }

    @Override
    public double calcularPerimetro() {
        return dimension1 * 3;
    }
}
